tests/e2e/lib.sh

```diff
@@ -454,9 +454,13 @@ check_for_errors_in_stackrox_logs() {
     # shellcheck disable=SC2010,SC2086
     filtered=$(ls $logs | grep -Ev "(previous|_describe).log$" || true)
     if [[ -n "$filtered" ]]; then
+        local check_out=""
         # shellcheck disable=SC2086
-        if ! scripts/ci/logcheck/check.sh $filtered; then
+        if ! check_out="$(scripts/ci/logcheck/check.sh $filtered)"; then
```

Comment:
[no-op] TIL this way to both capture the output and exit code.

Reply:
yes. Brad and I figured that out pair programming when running the script twice seemed lame.